test/extended/images/quota_admission.go

```diff
@@ -47,18 +38,48 @@ var _ = g.Describe("[images] openshift image resource quota", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 	})
 
-	g.It(fmt.Sprintf("should deny a push of built image exceeding quota"), func() {
+	g.AfterEach(func() {
+		g.By(fmt.Sprintf("Deleting quota %s", quotaName))
+		oc.AdminKubeREST().ResourceQuotas(oc.Namespace()).Delete(quotaName)
+
+		g.By("Deleting images")
+		iss, err := oc.AdminREST().ImageStreams(oc.Namespace()).List(kapi.ListOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+		for _, is := range iss.Items {
+			for _, history := range is.Status.Tags {
+				for i := range history.Items {
+					oc.AdminREST().Images().Delete(history.Items[i].Image)
+				}
+			}
+			for _, tagRef := range is.Spec.Tags {
+				switch tagRef.From.Kind {
+				case "ImageStreamImage":
+					nameParts := strings.Split(tagRef.From.Name, "@")
+					if len(nameParts) != 2 {
+						continue
+					}
+					imageName := nameParts[1]
+					oc.AdminREST().Images().Delete(imageName)
+				}
+			}
+			err := oc.AdminREST().ImageStreams(is.Namespace).Delete(is.Name)
+			o.Expect(err).NotTo(o.HaveOccurred())
+		}
+
+		g.By("Deleting shared project")
+		oc.AdminREST().Projects().Delete(oc.Namespace() + "-shared")
+	})
+
+	g.It("should deny a push of built image exceeding quota", func() {
```

Comment:
should deny a push of ~~built~~ image exceeding quota 
It's more clear, the built is confusing here.